src/solvers/strings/string_constraint_generator_float.cpp

```diff
@@ -278,7 +278,7 @@ std::pair<exprt, string_constraintst> add_axioms_for_fractional_part(
   // no_trailing_zero: forall j:[2, max_size[. !(|res| = j+1 && res[j] = '0')
   // a3 : int_expr = sum_j 10^j (j < |res| ? res[j] - '0' : 0)
 
-  const and_exprt a1(length_gt(res, 1), length_le(res, max));
+  const and_exprt a1(greater_than(res.length(), 1), length_le(res, max));
```

Comment:
:pick: use `{ }` initialization :bread:

Reply:
If we do that I think we should do it everywhere as a separate PR. So I'll leave it out. :butterfly: